README.md

```diff
@@ -23,22 +23,50 @@ You can build the project with all supported [Mbed OS build tools](https://os.mb
     $ mbed import mbed-os-example-for-aws && cd mbed-os-example-for-aws
     ```
 
+# Establishing AWS IoT Core service
+
+1. Create an AWS account if you don’t have one, and login to it.
+
+	**NOTE**: If you have an admin for your AWS account, please contact them to add a user to the account. You should obtain your login credentials from your admin in this case.
+
+
+2. Set up device credentials and policy via the AWS IoT console. You can refer to the AWS documentation [here](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html). After having logged into your AWS account. Follow the steps there to
+    1. Create an IoT policy
+     - Click on **IoT Core**
+      ![AWS_Core](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step1-IoT_Core.jpg)
+     - On the left panel click on **Secure > Policies**
+      ![AWS_Core](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/secure_policies_pic2.jpg)
+     - Click on **Create**
+      ![Create Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step7-create-policy.jpg)
+     - Click on **Advanced mode**
+      ![Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/creating_policy_click_advanced_mode.jpg)
+     - Fill in your policy as shown below then click **Create**
+      ![Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/aws_policy_advanced_mode_example.jpg)
+      **note:** you will need to substitute the **Resource** value above with your actual value from your **Thing** (which we will create next) by editing the policy after registering your **Thing**.
+      
+    2. Register a **Thing** 
+     - Click on **Manage > Things > Create**
+      ![Things](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step2-Things-Create.jpg)
+     - Click on **Create a single thing**
+      ![Create](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step3-create-single-thing.jpg)
+     - Fill in registry 
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/create_thing.jpg)
+      **note**: no need to specify a type
+      
+    3. Generate a device certificate and keys
+     - Click on **Create certificate**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/certificate_option.jpg)
+     - Download the public key, private key, and the certificate for this **Thing** then click on **Activate**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/download_certificates.jpg)
+     - Click on **Attach a policy**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/attach_policies.jpg)
+     - Ensure the correct policy is selected then click on **Register Thing**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/attach_policy_register_thing.jpg)
```

Comment:
Thanks for the detailed step-by-step guide. But it's effectively the same as this official page: https://docs.aws.amazon.com/iot/latest/developerguide/create-iot-resources.html. It's better not to maintain our own guide, as the service may change over time. Instead, maintain a link to an official one, and update the link as needed.

Reply:
I hadn't encountered that link before on the example docs. The links we had linking to AWS were [this ](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html)and [this](https://aws.amazon.com/iot-core/). But I agree that it would be helpful to include that new link on the docs. I suppose we could include the pictures so that people can quickly get things running by looking at the screenshots and if something has changed they could still roughly know what to look for or use the link. @LDong-Arm  What do you think?

Reply:
> I hadn't encountered that link before on the example docs. The links we had linking to AWS were [this ](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html)and [this](https://aws.amazon.com/iot-core/).

Yea the links we had previously were not helpful, so the new link is needed.

> But I agree that it would be helpful to include that new link on the docs. I suppose we could include the pictures so that people can quickly get things running by looking at the screenshots and if something has changed they could still roughly know what to look for or use the link. @LDong-Arm What do you think?

Pictures put extra maintenance on us, as the UI of AWS (as an external website) may change over time. The AWS doc (new link) is quite short and provides the exact steps incl. every button to click.

Reply:
We should update the link. I agree with @ARMmbed/billing-old that pictures put extra maintenance on us and we don't want to repeat what the AWS doc already describes.